docs/content/users/basics/developer-tools.md

```diff
@@ -38,42 +38,44 @@ To execute a fully-featured `composer create-project` command, you can execute t
 
-Note: if you run `ddev composer global require`, (or run `composer global require` inside the web container) the global packages will be installed in the in-container user's home directory ( ~/.composer) and will disappear on the next container restart, requiring rerun of the command. You may need an additional step of synchronizing created composer configuration and installed packages with the DDEV's [homeadditions folder](../extend/in-container-configuration.md) on the host.
+!!!tip "Careful with Global Requirements!"
+    If you run `ddev composer global require` (or `composer global require` inside the web container), global packages will be installed at the home directory within the container (`~/.composer`) and will disappear when the container restarts—meaning you’ll need to re-run the command.
+
+    You may want to synchronize created Composer configuration and installed packages with the DDEV’s [`homeadditions` directory](../extend/in-container-configuration.md) on your host machine.
 
 <a name="windows-os-and-ddev-composer"></a>
 
 ### Windows OS and `ddev composer`
 
-Both composer and some configurations of Docker Desktop for Windows introduce quite complex filesystem workarounds. DDEV attempts to help you with each of them.
-
-You generally don't have to worry about any of this, but it does keep things cleaner. Mostly just a few of the more complex TYPO3 projects have been affected.
+DDEV attempts to help with Composer and some configurations of Docker Desktop for Windows that introduce complex filesystem workarounds.
 
-* On some older configurations of Docker Desktop for Windows, symlinks are created in the container as "simulated symlinks", or XSym files. These are special text files that behave as symlinks inside the container (on CIFS filesystem), but appear as simple text files on the Windows host. (on the CIFS filesystem used by Docker for Windows inside the container there is no capability to create real symlinks, even though Windows now has this capability.)
-* DDEV-Local attempts to clean up for this situation. Since Windows 10/11+ (in developer mode) can create real symlinks, DDEV-Local scans your repository after a `ddev composer` command and attempts to convert XSym files into real symlinks. On older versions of Windows 10, it can only do this if your Windows 10 workstation is set to "Developer Mode".
-* On Windows 10/11+, to set your computer to developer mode, search for "developer" in settings. Screenshots are below.
+Aside from the occasional complicated TYPO3 project, you generally shouldn’t have to worry about any of this—it just keeps things cleaner:
```

Comment:
```suggestion
Use `ddev composer` (Composer inside the container) instead of using `composer` on the host side, because it uses the right version of PHP and all its extensions for your project:
```